keras_cv/models/classification/image_classifier_test.py

```diff
@@ -16,6 +16,7 @@
 
 import os
 
+import numpy as np
 import pytest
 import tensorflow as tf
 from absl.testing import parameterized
```

Comment:
(Line below this):

Should be `from keras_cv.backend import keras`

With that change, CI is passing on my local machine and we can merge.

Reply:
Done